src/pudl/extract/eia_bulk_elec.py

```diff
@@ -75,11 +75,12 @@ def _parse_data_column(elec_df: pd.DataFrame) -> pd.DataFrame:
             )
         else:
             data_df.loc[:, "date"] = pd.to_datetime(
-                data_df.loc[:, "date"], infer_datetime_format=True, errors="raise"
+                data_df.loc[:, "date"], errors="raise"
             )
         data_df["series_id"] = elec_df.at[idx, "series_id"]
         out.append(data_df)
     out = pd.concat(out, ignore_index=True, axis=0)
+    out = out.convert_dtypes()
```

Comment:
Tried changing this `convert_dtypes()` to `apply_pudl_dtypes()` and it failed, so leaving it like this.

Reply:
I don't think it's a big deal, but would it be worth investigating why `apply_pudl_dtypes()` is failing and fixing it, so we can consistently use `apply_pudl_dtypes`?